### PR TITLE
FIX: update completempu test for 1 MB limit

### PR DESCRIPTION
**Pull request by electrachong**
Needed to update this test now that MAX_POST_SIZE for POST route was updated in https://github.com/scality/S3/pull/560 to 1 MB.

### CODE REVIEW

File: tests/functional/aws-node-sdk/test/object/compluteMpu.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ const bucket = `bucketcompletempu-bucket-${Date.now()}`;
 
 const parts = [];
 
-// results in body of 589824 bytes
-for (let i = 0; i < 100000; i ++) {
+// results in body of 1049793 bytes (1 MB = 1048576)
+for (let i = 0; i < 15600; i ++) {
```
Comment (dora-korpar) [inline tests/functional/aws-node-sdk/test/object/compluteMpu.js]: how come this number changed?

Comment (electrachong) [inline tests/functional/aws-node-sdk/test/object/compluteMpu.js]: I think it makes sense to test a post size bigger than the limit but close to it.
Originally the number of parts being uploaded was 100,000 but this resulted in a post body size that was a lot larger than the limit, so even when we increased the limit the test was still passing.

**Comment by electrachong** [comment]
@ironman-machine

**Comment by rahulreddy** [comment]
@ironman-machine try

**Comment by rahulreddy** [comment]
Thanks @electrachong !

**Comment by rahulreddy** [comment]
Manually triggered end to end here http://ci.ironmann.io/gh/scality/Integration/9000#build-parameters

**Comment by rahulreddy** [comment]
End to end passed.

**Pull request merged**